test/parallel/test-tls-dhe.js

```diff
@@ -104,7 +104,11 @@ function testCustomParam(keylen, expectedCipher) {
   }, /DH parameter is less than 1024 bits/);
 
   // Custom DHE parameters are supported (but discouraged).
-  await testCustomParam(1024, dheCipher);
+  if (!common.hasOpenSSL(3, 2)) {
+    await testCustomParam(1024, dheCipher);
+  } else {
+    await testCustomParam(3072, dheCipher);
```

Comment:
Do we need to test 3072-bit keys? We're already testing 2048-bit keys on the line below. And if we do need to test 3072-bit keys, that could be with all version of OpenSSL?

Reply:
We tested with two different keys before so I kept it that way, we could add 3072 for all versions but that might add to runtime.  I think what is in the PR is my first choice but I don't feel to strongly one way or the other.

Reply:
I think testing the 3072-bit key will only slow down the test, but I also have no strong opionion.